packages/react-server/src/ReactFlightThenable.js

```diff
@@ -22,6 +22,15 @@ import type {
 
 export opaque type ThenableState = Array<Thenable<any>>;
 
+// An opaque object that is thrown (e.g. by `use`) to trigger Suspense. If we
+// detect this is caught by userspace, we'll log a warning in development.
+// TODO: Alternatively, this could be an actual error with a helpful message.
+// The reason not to do that is because someone might be tempted to check the
+// message in order to determine if it's caused by Suspense. Maybe it could be
+// an actual error object in production, but a plain object in development,
+// where it would also be accompanied by a warning.
+export const SuspenseException: mixed = {};
```

Comment:
Maybe make it `new (function SuspenseException(){})` so that it gets a "class name" in the debug tools. I guess it would get minified anyway unfortunately.

Reply:
Why not explicitly assign .name?

Reply:
You can't in older browsers. That's new.

Reply:
I'll change it to be an error for now, seems like the least bad option